### PR TITLE
Netherite items

### PR DESCRIPTION
**Pull request by kaekaes**
You can use modifiers on netherite items.
Knockback resistance is added to armor values (but i'm not sure it works well)

### CODE REVIEW

**Comment by kaekaes** [comment]
> This changes going to break support for older versions.

Sure, I'll change that 😉

File: src/main/java/de/flo56958/minetinker/commands/subs/NameCommand.java [hunk 1]
```diff
@@ -1,127 +1,127 @@
...
+			CommandManager.sendError(sender, LanguageManager.getString("Commands.Failure.Cause.PlayerMissing"));
+			return true;
+		}
+		ItemStack tool = player.getInventory().getItemInMainHand();
+
+		if (ModManager.instance().isToolViable(tool) || ModManager.instance().isArmorViable(tool)) {
+			StringBuilder name = new StringBuilder();
+
+			for (int i = 1; i < args.length; i++) {
+				name.append(" ").append(args[i].replace('&', ''));
```
Comment (Flo56958) [inline src/main/java/de/flo56958/minetinker/commands/subs/NameCommand.java:51]: Why is that changed? it is necessary for the rename command to work properly

File: src/main/java/de/flo56958/minetinker/modifiers/ModManager.java [hunk 2]
```diff
@@ -1,1137 +1,1170 @@
...
+				return Material.getMaterial("WOODEN_" + tool);
+			case COBBLESTONE:
+				return Material.getMaterial("STONE_" + tool);
+			case IRON_INGOT:
+				return Material.getMaterial("IRON_" + tool);
+			case GOLD_INGOT:
+				return Material.getMaterial("GOLDEN_" + tool);
+			case DIAMOND:
+				return Material.getMaterial("DIAMOND_" + tool);
+			case NETHERITE_INGOT:
```
Comment (Flo56958) [inline src/main/java/de/flo56958/minetinker/modifiers/ModManager.java:229]: That will unfortunately break in 1.15.2 as NETHERITE_INGOT is not available. Youll need to check for 1.16 first and then check for the material separate

File: src/main/java/de/flo56958/minetinker/modifiers/ModManager.java [hunk 3]
```diff
@@ -1,1137 +1,1170 @@
...
+		switch (material) {
+			case LEATHER:
+				return Material.getMaterial("LEATHER_" + tool);
+			case IRON_INGOT:
+				return Material.getMaterial("IRON_" + tool);
+			case GOLD_INGOT:
+				return Material.getMaterial("GOLDEN_" + tool);
+			case DIAMOND:
+				return Material.getMaterial("DIAMOND_" + tool);
+			case NETHERITE_INGOT:
```
Comment (Flo56958) [inline src/main/java/de/flo56958/minetinker/modifiers/ModManager.java:246]: Same here

File: src/main/java/de/flo56958/minetinker/modifiers/ModManager.java [hunk 4]
```diff
@@ -1,1137 +1,1170 @@
...
+				toughness = 2.0d;
+				break;
+			case CHAINMAIL_LEGGINGS:
+				armor = 4.0d;
+				break;
+			case DIAMOND_LEGGINGS:
+				armor = 6.0d;
+				toughness = 2.0d;
+				break;
+			case NETHERITE_HELMET:
```
Comment (Flo56958) [inline src/main/java/de/flo56958/minetinker/modifiers/ModManager.java:955]: same here

File: contributors.json [hunk 5]
```diff
@@ -1,60 +1,68 @@
...
+    "transifex": null,
+    "transifex_languages": [],
+    "other": [ "Modifier Inventor" ]
+  },
+  {
+    "github": "kaekaes",
+    "discord": "AtheroX#8624",
+    "minecraft": "99be3c12-049b-4196-9cf9-8312f88e1728",
+    "transifex": "kaekaes",
+    "transifex_languages": [ "en_US", "es_ES", "ca_ES" ],
```
Comment (Flo56958) [inline contributors.json:65]: here should only be the languages listed you actively work on and that are available on transifex.

File: src/main/java/de/flo56958/minetinker/commands/subs/NameCommand.java (hunk 1 above)
Comment (kaekaes) [reply src/main/java/de/flo56958/minetinker/commands/subs/NameCommand.java:51]: The change was §. It seems like it wasn't in UTF-8 or something

File: src/main/java/de/flo56958/minetinker/modifiers/ModManager.java (hunk 2 above)
Comment (kaekaes) [reply src/main/java/de/flo56958/minetinker/modifiers/ModManager.java:229]: Ok, i'll do that

File: contributors.json (hunk 5 above)
Comment (kaekaes) [reply contributors.json:65]: Ok ^^
